osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -124,9 +125,27 @@ public abstract partial class SongSelect : ScreenWithBeatmapBackground, IKeyBind
         [Resolved]
         internal IOverlayManager? OverlayManager { get; private set; }
 
+        private Bindable<float> backgroundBlurLevel { get; set; } = new BindableFloat();
+
+        private void applyBackgroundBlur(float v)
+        {
+            ApplyToBackground(background =>
+            {
+                background.IgnoreUserSettings.Value = true;
+                background.BlurAmount.Value = v * BACKGROUND_BLUR;
+            });
+        }
+
+        private void applyBackgroundBlur(ValueChangedEvent<float> v)
+        {
+            applyBackgroundBlur(v.NewValue);
```

Comment:
you can set this in the bindvaluechanges why is this its own method im incredibly confused here check any usage of .BindValueChanged  to see how this should be done

Reply:
the use of an intermediate method was used for unsubscribing to the event when needed